x-pack/plugins/profiling/server/routes/setup.ts

```diff
@@ -126,13 +129,12 @@ export function registerSetupRoute({
           logger,
           packagePolicyClient: dependencies.start.fleet.packagePolicyService,
           soClient: core.savedObjects.client,
-          spaceId: dependencies.setup.spaces.spacesService.getSpaceId(request),
+          spaceId:
```

Comment:
__note:__ just a note, it's unfortunate that `Fleet` requires Space ID while it doesn't have to, but, I agree, it's easier/safer to fall back to DEFAULT_SPACE_ID than rewriting all fleet's methods signatures.